src/main/java/bio/terra/service/filedata/flight/ingest/IngestFileDirectoryStep.java

```diff
@@ -16,8 +16,15 @@
 import bio.terra.stairway.StepResult;
 import bio.terra.stairway.StepStatus;
 import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.atomic.AtomicLong;
 
 public class IngestFileDirectoryStep implements Step {
+    private static final Logger logger = LoggerFactory.getLogger(IngestFileDirectoryStep.class);
+    private static AtomicLong abortCounter = new AtomicLong();
```

Comment:
When will this static variable be reset? And I think it will be shared by two concurrent ingest flights, even if they are ingesting into different directories -- is that your intent?

Reply:
Removed this. I meant it just for debugging. When we have a way to log where we can harvest the data in BQ, we can log this event and count it that way.